DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: tinysnapshot
 Type: Package
 Title: Snapshots for Unit Tests using the 'tinytest' Framework
-Version: 0.0.7
+Version: 0.0.7.1
```

Comment:
Same here :)   Didn't want to impose my style on working difference though.  Nice to see the same pattern.